docusaurus/docs/dev-docs/backend-customization/controllers.md

```diff
@@ -184,24 +198,30 @@ export default {
 When a new [content-type](/dev-docs/backend-customization/models#content-types) is created, Strapi builds a generic controller with placeholder code, ready to be customized.
 :::
 
-### Sanitization in controllers
+### Sanitization and Validation in controllers
 
 :::warning
-As of Strapi v4.8.0 and greater it's strongly recommended you sanitize your incoming request query and parameters utilizing the new `sanitizeQuery` function to prevent leaking of private data.
+It's strongly recommended you sanitize (v4.8.0) and/or validate (v4.13.0) your incoming request query and parameters utilizing the new `sanitizeQuery` and `validateQuery` functions to prevent leaking of private data.
 :::
 
 #### Sanitization when utilizing controller factories
 
-Within the Strapi factories there are 3 functions exposed that can be used for sanitization:
+Within the Strapi factories the following functions are exposed that can be used for sanitization and validation:
 
 | Function Name    | Parameters                 | Description                                                                          |
 |------------------|----------------------------|--------------------------------------------------------------------------------------|
 | `sanitizeQuery`  | `ctx`                      | Sanitizes the request query                                                          |
 | `sanitizeOutput` | `entity`/`entities`, `ctx` | Sanitizes the output data where entity/entities should be an object or array of data |
 | `sanitizeInput`  | `data`, `ctx`              | Sanitizes the input data                                                             |
+| `validateQuery`  | `ctx`                      | Validates the request query (throws an error on invalid params)                      |
+| `validateInput`  | `data`, `ctx`              | Validates the input data (throws an error on invalid data)                           |
 
 These functions automatically inherit the sanitization settings from the model and sanitize the data accordingly based on the content-type schema and any of the content API authentication strategies, such as the Users & Permissions plugin or API tokens.
 
+:::warning
+Because these methods use the model associated with the current controller, if you query data that is from another model (ie, doing a find for "menus" within a "restaurant" controller method) you must instead use the `@strapi/utils` tools, such as `sanitize.contentAPI.query` described below, or else the result of your query will be sanitized against the wrong model.
```

Comment:
```suggestion
Because these methods use the model associated with the current controller, if you query data that is from another model (i.e., doing a find for "menus" within a "restaurant" controller method), you must instead use the `@strapi/utils` tools, such as `sanitize.contentAPI.query` described below, or else the result of your query will be sanitized against the wrong model.
```